SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2739,7 +2739,10 @@ void USpatialReceiver::QueueAuthorityOpForAsyncLoad(const Worker_ComponentSetAut
 {
 	EntityWaitingForAsyncLoad& AsyncLoadEntity = EntitiesWaitingForAsyncLoad.FindChecked(Op.entity_id);
 
-	AsyncLoadEntity.PendingOps.SetAuthority(Op.entity_id, Op.component_set_id, static_cast<Worker_Authority>(Op.authority));
+	// todo this needs to be changed as it abuses the authority ops in a way that happens to work here.
```

Comment:
I don't understand the reasoning here - but okay if it'll be resolved with the pending proper component set usage

Reply:
It gets solved with the actor system. Currently the async asset loading tries to record all ops that get sent to it. However when we upgraded to 15 we didn't change how anything using "ops" interacted with authority delegations. This is sort of cancelled out by also not changing how we convert view deltas to op lists for legacy code. So this function call is one that would normally be cause for concern.

So the fixes are to stop using ops for things that aren't really ops, or to turn this into a completeness filter, which is what actor systems will eventually do.